plugins/usdc_coingecko/usdc_coingecko_test.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/stretchr/testify/require"
+	"log"
+	"testing"
+)
+
+func TestNewCoinGeckoClient(t *testing.T) {
+	client := NewCoinGeckoClient(&defaultConfig)
+	prices, err := client.FetchPrice([]string{"USDC-USD"})
+	require.NoError(t, err)
+	log.Println(prices)
```

Comment:
Same comment as above, perhaps a require instead of a log.

Reply:
Resolved in 4d25fa7c661e76eeaac09170f91d4d6d438fd7ea